.github/workflows/ci.yml

```diff
@@ -85,7 +85,7 @@ jobs:
             SEP=","
           done
 
-          ./mvnw test -pl "$PROJECTS" -am
+          ./mvnw install -pl "$PROJECTS" -am
```

Comment:
Did this command have to be replaced because of the the downgrade, or is there another motivation?

Reply:
Yes, it's explained in the description of the PR. It seems that doing `test` the multi-release is not detected. My feeling is that the classpath when testing is using the `core/target/classes` (to access core classes) which has no `MANIFEST.MF` and it is not multi-release. Doing the `install` it uses the jar which is OK. I don't know how this worked before, probably a different maven version now.

Reply:
Gotcha, thanks for the explainer.